.gitignore

```diff
@@ -58,3 +58,6 @@ node_modules
 # Ignore build for client
 /client/build
 .DS_Store
+
+#Ignore local node version
```

Comment:
Add a space behind this `#`.

Also, should this be PRed separately? Should be a relatively safe PR that we can merge early and will improve dev experience.

Reply:
Good suggestion! Created a new PR here:
https://github.com/Coursemology/coursemology2/pull/4030